### ISSUE TITLE
Docs: 404

### ISSUE THREAD

**Issue by m-a-saleh** [aiidateam/aiida-core#5035]
New documentations not found.

Examples:

This page:
https://aiida.readthedocs.io/projects/aiida-core/en/v1.0.0/_modules/aiida/backends/tests/orm/utils/test_loaders.html
directs to a newer version, which is non-existing:
https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.4/_modules/aiida/backends/tests/orm/utils/test_loaders.html

This page:
https://aiida.readthedocs.io/projects/aiida-core/en/v0.12.1/_modules/aiida/orm/data/parameter.html
directs to a newer version, which is non-existing:
https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.4/_modules/aiida/orm/data/parameter.html

This page:
https://aiida.readthedocs.io/projects/aiida-core/en/v0.9.1/orm/dev.html
directs to a newer version, which is non-existing:
https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.4/orm/dev.html

**Comment by ltalirz** [comment]
thanks for the report @m-a-saleh 

Indeed, over time there've been several significant documentation overhauls, and some parts of the doc are missing or in different places.

For docs that have moved, redirects can be added simply in the following file:
https://github.com/aiidateam/aiida-core/blob/develop/docs/source/redirects.txt

Would you be willing to make a small PR that redirects some of these pages to the appropriate place in the new docs?

**Comment by giovannipizzi** [comment]
Adding also https://aiida.readthedocs.io/projects/aiida-core/en/v1.0.0/concepts/provenance.html does not work in the newest version: https://aiida.readthedocs.io/projects/aiida-core/en/v2.0.0b1/concepts/provenance.html and should be added to the redirects.
In addition, the 404 page at the moment seems to have a very weird layout, probably some wrong CSS path or something similar: 

<img width="1772" alt="Screenshot 2022-03-22 at 22 04 03" src="https://user-images.githubusercontent.com/16132028/159575756-bdd25d50-0319-44be-b966-47b3e4495d34.png">

**Comment by giovannipizzi** [comment]
@chrisjsewell are you OK to add this to the 2.0 milestone? (ideally checking if there are more broken links - is there some easy way to search for online links pointing to old documentation, so we can check/

**Comment by giovannipizzi** [comment]
A few more:
- https://aiida.readthedocs.io/projects/aiida-core/en/v1.0.0/concepts/calculations.html should point to https://aiida.readthedocs.io/projects/aiida-core/en/v1.6.7/topics/calculations/concepts.html

- https://aiida-core.readthedocs.io/en/v1.0.0/querying/querybuilder/intro.html should be now https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/database.html#advanced-querying

**Comment by sphuber** [comment]
As mentioned, the docs have been overhauled significantly and some of the problematic links date to very old versions. On top of that, these are pages from eh API pages (of which there are many) and so adding all of these to an automated redirect will be a lot of work. Since this has been open for a long time, I think we can close this for now.

**Comment by giovannipizzi** [comment]
I don't think we should close this without fixing. Old links are still around on the web, and at least my redirects should be implemented before closing this issue

**Comment by Tezas-6174** [comment]
Is this issue solved? If not, I'd like to work on it.

**Comment by ltalirz** [comment]
Cheers @Tezas-6174 , further help on this is always welcome!

**Comment by Tezas-6174** [comment]
Thank you! So, what are the tasks that are needed to complete this issue?

- [ ] The pages mentioned above should direct to the correct version page.

What else? Can you elaborate more if I am missing something?

**Comment by ltalirz** [comment]
> So, what are the tasks that are needed to complete this issue?

This:

> > Old links are still around on the web, and at least my redirects should be implemented before closing this issue

I.e. at least implement the redirects mentioned by Giovanni; 
ideally also check the top results for aiida documentation found on search engines and make sure they work / redirect them to a sensible newer version of the documentation.


> Also, let me know the approach that is expected.

this

> > For docs that have moved, redirects can be added simply in the following file:
> > https://github.com/aiidateam/aiida-core/blob/develop/docs/source/redirects.txt

**Comment by Tezas-6174** [comment]
Ok. I got it and am working on it.

**Comment by m-a-saleh** [comment]
@0xt3j4s I can help if my help is needed

**Comment by ganesh17091** [comment]
Hi! I would like to work on this issue as my first contribution to AiiDA. Is it still available?
